packages/firebase/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import "@dzangolab/fastify-mercurius";
+import { verifySession } from "supertokens-node/recipe/session/framework/fastify";
+
+import notificationHandlers from "./model/notification/handlers";
+import deviceHandlers from "./model/userDevice/handlers";
+import { User } from "./types";
+
+declare module "fastify" {
+  interface FastifyInstance {
+    verifySession: typeof verifySession;
+  }
+}
+
+declare module "mercurius" {
+  interface MercuriusContext {
+    user: User;
+  }
+}
+
+declare module "@dzangolab/fastify-config" {
+  interface ApiConfig {
+    firebase?: {
+      projectId: string;
+      privateKey: string;
+      clientEmail: string;
+      table?: {
+        userDevices?: {
+          name: string;
+        };
+      };
+    };
+    notification?: {
```

Comment:
could you make it similar to this https://github.com/dzangolab/fastify/blob/main/packages/mailer/src/types.ts#L13?